packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,402 @@
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="end" direction="bottom">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="end" direction="top">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Autofocus
+
+```tsx example
+<MenuTrigger align="start" closeOnSelect={false}>
+  <ActionButton>
+      Menu Button Autofocus
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    autoFocus>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="start" closeOnSelect={false}>
+  <ActionButton>
+      Menu Button Autofocus Last
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    autoFocus="last">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Closes Menu onSelect
+
+```tsx example
+<MenuTrigger align="start" closeOnSelect={false}>
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Disabled Trigger
+
+```tsx example
+<MenuTrigger align="start">
+  <ActionButton isDisabled>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Disabled Menu Items
+
+```tsx example
+<MenuTrigger align="start">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    disabledKeys={['Aardvark', 'Snake']}>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="start">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark', key: 1}, {name: 'Kangaroo', key: 2}, {name: 'Snake', key: 3}]}
+    itemKey="name"
+    disabledKeys={['1', '3']}>
+    {item => <Item key={item.key}>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Open
+
+```tsx example
+<MenuTrigger align="start" isOpen>
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="start" defaultOpen>
```

Comment:
This example isn't great because any blur or scroll closes it, which the element next to it causes.

Reply:
I agree, neither of these is great really. By the time you scroll down to the default open one, it's closed.
And the isOpen one has flipped and is in the way of the code.
Maybe no example for this one? just a description?